docs/mirror-bitbucket-server.md

```diff
@@ -0,0 +1,20 @@
+# Mirroring Bitbucket Server organizations and repos in Snyk
```

Comment:
this should not be in this PR, when we merge this the mirroring is not yet available until we merge all the commands

Reply:
Done